deku-c/python-client/README.md

```diff
@@ -0,0 +1,39 @@
+# Python Deku Client Library
```

Comment:
Can we just link to the docs from this page? Less chance of failing to keep the two docs in sync.